File: conexiones/conexion.php/mongo.php
<?php
require 'vendor/autoload.php';

// Crear la conexión a MongoDB
$cliente = new MongoDB\Client("mongodb://localhost:27017");

// Imprimir la conexión y verificar si está exitosa
echo "Conexión exitosa\n";

// Imprimir el tipo de objeto de la conexión
echo "Tipo de conexión: " . get_class($cliente) . "\n";

// Listar las bases de datos disponibles
echo "Bases de datos disponibles: \n";
$dbs = $cliente->listDatabases();
foreach ($dbs as $db) {
    echo $db->getName() . "\n";
}
?>

File: conexiones/conexion.java/mongo.java
import com.mongodb.MongoClient;
import com.mongodb.MongoClientURI;
import com.mongodb.client.MongoDatabase;

public class ConexionMongoDB {
    public static void main(String[] args) {

        String uri = "mongodb://localhost:27017/";
        
        MongoClient cliente = new MongoClient(new MongoClientURI(uri));
        
        // Imprimir el cliente de conexión y el tipo de conexión
        System.out.println(cliente);
        System.out.println("Conexión exitosa");
        System.out.println("Tipo de conexión: " + cliente.getClass().getName());
        
        // Listar las bases de datos disponibles
        System.out.println("Bases de datos disponibles:");
        for (String dbName : cliente.listDatabaseNames()) {
            System.out.println(dbName);
        }

        // Cerrar la conexión
        cliente.close();
    }
}
